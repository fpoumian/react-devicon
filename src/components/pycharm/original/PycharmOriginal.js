import React from 'react'
import PropTypes from 'prop-types'
import SVGDeviconInline from '../../_base/SVGDeviconInline'

import iconSVG from './PycharmOriginal.svg'

/** PycharmOriginal */
function PycharmOriginal({ width, height, className }) {
  return (
    <SVGDeviconInline
      className={'PycharmOriginal' + ' ' + className}
      iconSVG={iconSVG}
      width={width}
      height={height}
    />
  )
}

PycharmOriginal.propTypes = {
  className: PropTypes.string,
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
}

export default PycharmOriginal
