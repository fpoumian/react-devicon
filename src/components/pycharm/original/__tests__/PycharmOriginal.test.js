import React from 'react'
import renderer from 'react-test-renderer'
import PycharmOriginal from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<PycharmOriginal />).toJSON()
  expect(tree).toMatchSnapshot()
})
