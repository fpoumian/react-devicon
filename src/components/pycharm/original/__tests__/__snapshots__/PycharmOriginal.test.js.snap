// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="SVGInline PycharmOriginal undefined"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon PycharmOriginal-devicon undefined-devicon\\" id=\\"pycharm\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 128 128\\"><defs><style>.cls-1{fill:#fa930d}.cls-2{fill:#4d8548}</style></defs><title>Artboard 14</title><g id=\\"original\\"><path class=\\"cls-1\\" d=\\"M96.25 40.67h-.51a23.45 23.45 0 0 0-22.86 17.57h-.12a20.1 20.1 0 0 1-39.85-3.78 15.9 15.9 0 0 1 .16-2.21c0-.15 0-.3.06-.45s.1-.57.16-.85A63.63 63.63 0 0 1 66.75 7.43a48.94 48.94 0 0 0-62.54 42.8c-.06.18-.15.35-.21.53v71.55h29V99.18a49 49 0 0 0 67-58.51h-3.8z\\"/><path class=\\"cls-2\\" d=\\"M67.92 57.32l.09-.15A28.23 28.23 0 0 1 95.4 35.75h13.85a15 15 0 0 0 6.16-1.51c-.48-.06-1-.13-1.44-.23a19.82 19.82 0 0 1-15-13.41A19.83 19.83 0 0 0 110.8 28a19.64 19.64 0 0 0 11.88-1.35 15.13 15.13 0 0 0-13.82-21.17H95.4a58.54 58.54 0 0 0-57.18 46.04 15 15 0 0 0-.39 3 15.18 15.18 0 0 0 30.09 2.86zm36.23-43.1a4.7 4.7 0 1 1 3.69 5.52 4.71 4.71 0 0 1-3.68-5.52zM124 107.4a15.15 15.15 0 0 1-15.15 15.15H95.4a58.23 58.23 0 0 1-38.3-14.34A53.72 53.72 0 0 0 91.72 92a29.19 29.19 0 0 0 3.68.26h13.45A15.16 15.16 0 0 1 124 107.4z\\"/></g></svg>",
    }
  }
/>
`;
